nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -656,6 +658,19 @@ where
             <RuntimeServiceId as AsServiceId<Self>>::SERVICE_ID
         );
 
+        wait_until_services_are_ready!(
+            &self.service_resources_handle.overwatch_handle,
+            Some(Duration::from_millis(3000)),
+            NetworkService<_, _>,
+            BlendService<_, _, _>,
+            TxMempoolService<_, _, _>,
+            DaMempoolService<_, _, _, _, _, _, _, _, _, _, _>,
+            DaSamplingService<_, _, _, _, _, _, _, _, _>,
+            StorageService<_, _>,
+            TimeService<_, _>
+        )
+        .await?;
```

Comment:
I would separate network related services from others. Always waiting for network ones first. Then the rest of them. Just because network is actually crucial for any service that depend on it.

Reply:
Is it because you want to specify a different timeout for those? Otherwise I think it matters not. 
As long as one fails, the service won't startup. Which applies the same for `Network` as for any other service. It doesn't really matter if you split the calls.

Reply:
Yeah, I was thinking about using different timings. But if we go for the simple long wait it should be ok to keep then together.